src/balena.sbvr

```diff
@@ -443,6 +445,10 @@ Fact type: application has uuid
 	Necessity: each application has exactly one uuid.
 	Necessity: each uuid is of exactly one application.
 	Necessity: each application has a uuid that has a Length (Type) that is equal to 32.
+Fact type: application is public
+Fact type: application has install type
+	Necessity: each application has at most one install type
```

Comment:
Should this be
```suggestion
	Necessity: each application has exactly one install type
```
since there's a default?

Reply:
updated